src/hooks/useClipboard.ts

```diff
@@ -3,11 +3,10 @@ import { useCallback, useEffect, useState } from 'react';
 import useAppState from './useAppState';
 import { deviceUtils } from '@rainbow-me/utils';
 
-const listeners = new Set();
+const listeners = new Set<React.Dispatch<React.SetStateAction<string>>>();
```

Comment:
Not sure what is our convention, but maybe add `Dispatch` and `SetStateAction` to import from `react`? Would be much shorter.